hack/make/sign-repos

```diff
@@ -26,7 +26,7 @@ sign_packages(){
 
 		# sign the repo metadata
 		for F in $(find $APTDIR -name Release); do
-			if test "$F" -nt "$F.gpg" ; then
+			if ! test -e "$F.gpg" || test "$F" -nt "$F.gpg" ; then
```

Comment:
Doesn't `-nt` already "do the right thing" here?  If `$F.gpg` doesn't exist, `$F` should be "newer":
```console
$ ls
$ touch a
$ test a -nt b && echo yes || echo no
yes
$ test b -nt a && echo yes || echo no
no
```